pkgs/build-support/xen/default.nix

```
@@ -254,8 +257,8 @@ stdenv.mkDerivation (finalAttrs: {
     mkdir --parents $out $out/share $boot
     cp -prvd dist/install/nix/store/*/* $out/
     cp -prvd dist/install/etc $out
+    find dist/install/boot -type f -name '*.gz' -print -exec gunzip -k '{}' ';'
```

```suggestion
    # Unzip the compressed multiboot image so Limine picks it up in the module.
    find dist/install/boot -type f -name '*.gz' -print -exec gunzip -k '{}' ';'
```
Because I *will* forget what this does.